src/pages/SubmissionDetails.js

```diff
@@ -116,9 +116,8 @@ export const SubmissionDetails = ({ submissionId }) => {
     }
 
     loadRunsData()
```

Comment:
Assuming we don't need runs data to load run_Set_Data, I would fetch runSet data first.

Imagine you had 100,000 runs... it's probably better to show the top level run set details while the runs are still loading in rather than delay the high level details behind loading all the runs into memory

Reply:
good thinking, thanks!